.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
```

Comment:
Overall its written very well, and clean! Again, please add few tests, at least basic ones, and in addition, I'll need time to test it to see if it works and give you my feedback. Good job!